pkg/controllers/multiclusterservice/mcs_controller.go

```diff
@@ -337,8 +338,8 @@ func (c *MCSController) retrieveService(ctx context.Context, mcs *networkingv1al
 		delete(svcCopy.Labels, networkingv1alpha1.MultiClusterServicePermanentIDLabel)
 	}
 
-	if err = c.Client.Update(ctx, svcCopy); err != nil {
```

Comment:
Is there a sign that here conflicts offen?
I suppose the Service won't be updated very frequently, right?